1337.py

```diff
@@ -0,0 +1,13 @@
+class Solution:
+    def kWeakestRows(self, mat: List[List[int]], k: int) -> List[int]:
+        rows, res = [], []
+
+        for i in range(len(mat)):
+            rows.append((Counter(mat[i])[1], i))
+
+        rows = sorted(rows, key=lambda x : x[0])
```

Comment:
If the key is simply `x[0]`, you probably don't need to explicitly specify it in the `key=` argument. Sorting, by default, uses the first element in a `tuple` as the key.